pkgs/by-name/zo/zotero-beta/package.nix

```diff
@@ -122,15 +95,28 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
     ln -s "$prefix/usr/lib/zotero-bin-${version}/zotero" "$out/bin/"
 
-    # install desktop file and icons.
+    # Install desktop file and icons.
     mkdir -p $out/share/applications
     cp ${desktopItem}/share/applications/* $out/share/applications/
     for size in 32 64 128; do
       install -Dm444 icons/icon$size.png \
         $out/share/icons/hicolor/''${size}x''${size}/apps/zotero.png
     done
     install -Dm444 icons/symbolic.svg \
-        $out/share/icons/hicolor/symbolic/apps/zotero-symbolic.svg
+      $out/share/icons/hicolor/symbolic/apps/zotero-symbolic.svg
+
+    for executable in \
```

Comment:
Patchelfing of output binaries should happen in fixup, regardless of whether it's automatic via `autoPatchelfHook` or manual